rerun_bridge/src/rerun_bridge/rerun_ros_interface.cpp

```diff
@@ -354,7 +365,12 @@ void log_point_cloud2(
             std::memcpy(&position.xyz.xyz[0], &msg->data[point_offset + x_offset], sizeof(float));
             std::memcpy(&position.xyz.xyz[1], &msg->data[point_offset + y_offset], sizeof(float));
             std::memcpy(&position.xyz.xyz[2], &msg->data[point_offset + z_offset], sizeof(float));
-            points.emplace_back(std::move(position));
+            if (has_rgb) {
+                uint8_t rgba[4];
+                std::memcpy(&rgba, &msg->data[point_offset + rgb_offset], sizeof(uint32_t));
+                colors.emplace_back(rerun::Color(rgba[2], rgba[1], rgba[0]));
```

Comment:
It looks like this does BGRA -> RGB? So maybe 

```suggestion
                uint8_t bgra[4];
                std::memcpy(&bgra, &msg->data[point_offset + rgb_offset], sizeof(uint32_t));
                colors.emplace_back(rerun::Color(bgra[2], bgra[1], bgra[0]));
```